### DISCUSSION TITLE
Wagmi won't install

### DISCUSSION THREAD

**Discussion by woodjs** [wevm/wagmi#1805] [Q&A]
I am trying to install the wagmi library but I am getting a trail. error:
```
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent An unknown git error occurred
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```
I also tried to install open projects that have wagmi dependencies, I get exactly the same error.
What solutions are there?

**Comment by tmm** [comment]
What command are you running/package manager?

**Reply by woodjs** [reply]
npm i wagmi / NPM

**Comment by woodjs** [comment]
It's pretty dumb, but the solution is to install Git and run the code editor as an administrator. It helped me to install this library.
